animation-worklet/anim-worklet-worker.js

```diff
@@ -26,14 +26,23 @@ limitations under the License.
     }
 
     get top() { return this.proxy_.scrollTop; }
+    set top(s) { this.proxy_.scrollTop = s;}
     get left() { return this.proxy_.scrollLeft; }
+    set left(s) { this.proxy_.scrollLeft = s;}
+
   };
 
   class ProxyElementWrapper {
     constructor(desc) {
+      this.proxy_ = desc.proxy;
       this.scrollOffsets = new ScrollOffsetsWrapper(desc.proxy);
       this.styleMap = new StyleMapWrapper(desc);
     }
+
+    get ready() {
```

Comment:
Thanks for fixing this, I don't see ready() as part of the proposal at https://github.com/majido/animation-worklet-proposal), can we suffix this with "_" to imply that it is private? See https://google.github.io/styleguide/javascriptguide.xml?showone=Naming#Naming